rpc_jobs/standard_job_premerge.yml

```diff
@@ -138,13 +139,14 @@
                   }} // withCredentials
                 }} // dir
               }} // ansiColor
-            }} catch (e) {{
-              print(e)
-              currentBuild.result="FAILURE"
-              throw e
             }} finally {{
+              dir(env.RE_HOOK_RESULT_DIR){{
```

Comment:
Can we just add this to ``archive_artifacts`` so that we don't need to add it to every job?

Reply:
Done.